pkg/asset/installconfig/azure/validation.go

```diff
@@ -215,12 +235,26 @@ func validateInstanceTypes(client API, ic *types.InstallConfig) field.ErrorList
 			if ultraSSDCapability == "" {
 				ultraSSDCapability = defaultUltraSSDCapability
 			}
+			capabilities, err = client.GetVMCapabilities(context.TODO(), instanceType, ic.Azure.Region)
+			if err != nil {
+				allErrs = append(allErrs, field.Invalid(fieldPath.Child("platform", "azure", "type"), instanceType, err.Error()))
+			}
 			if vmNetworkingType == "" {
-				vmNetworkingType = defaultVMNetworkingType
+				val, ok := capabilities[string(azure.AcceleratedNetworkingEnabled)]
```

Comment:
Same problem as in the controlPlane: capabilities can be `nil`.

Reply:
Fixed! thanks